packages/kotti-ui/source/kotti-field-text-area/KtFieldTextArea.vue

```diff
@@ -27,9 +31,26 @@ export default defineComponent({
 			props,
 			supports: KOTTI_FIELD_TEXT_AREA_SUPPORTS,
 		})
+		const textareaRef = ref<HTMLTextAreaElement | null>(null)
 
 		const { forceUpdate, forceUpdateKey } = useForceUpdate()
 
+		const resizeTextarea = (
+			textarea: HTMLTextAreaElement | null,
+			autoSize: boolean,
+			maxHeightInPixels: number | null,
+		) => {
+			if (
+				autoSize &&
+				textarea &&
+				((maxHeightInPixels && textarea?.scrollHeight < maxHeightInPixels) ||
+					!maxHeightInPixels)
+			) {
+				textarea?.setAttribute('style', 'height: auto')
+				textarea?.setAttribute('style', `height: ${textarea.scrollHeight}px`)
```

Comment:
Again, you don't need the optional chaining operators `?.`, because you are already checking `textarea` in line 45